dependency_manager/src/edm_tool/edm.py

```diff
@@ -19,6 +19,7 @@
 import requests
 import re
 import datetime
+from edm_tool import bazel
```

Comment:
in general it might be good to leave a blank line between system imports and local imports for better grouping and readability (as some formatters rearange them alphabetically)

Reply:
OK